client/me/help/main.jsx

```diff
@@ -3,34 +3,32 @@
 /**
  * External dependencies
  */
-
-import { find } from 'lodash';
-import { localize } from 'i18n-calypso';
-import React from 'react';
 import debugModule from 'debug';
+import React from 'react';
```

Comment:
why are you reordering the React import?

Reply:
I reordered the imports alphabetically.